src/modules/facility-fixed-fee/dto/get-facility-fixed-fee-response.dto.ts

```diff
@@ -0,0 +1,100 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { EXAMPLES, PROPERTIES } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers';
+
+export type GetFacilityFixedFeeResponse = GetFacilityFixedFeeResponseItem[];
+export class GetFacilityFixedFeeResponseItem {
+  @ApiProperty({
+    description: 'The identifier of the facility.',
+    example: EXAMPLES.FACILITY_ID,
+    minLength: 10,
+    maxLength: 10,
```

Comment:
You can use just use `length: 10` for this

Reply:
My bad, I was thinking of the custom decorators we implemented for which you can use just `length`, but this one is nestjs/swagger